packs/equipment/channel-protection-amulet.json

```diff
@@ -37,7 +37,32 @@
             "title": "Pathfinder GM Core"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "definition": [
+                    {
+                        "or": [
+                            {
+                                "and": [
+                                    "self:mode:living",
+                                    "item:slug:harm"
+                                ]
+                            },
+                            {
+                                "and": [
+                                    "self:mode:undead",
+                                    "item:slug:heal"
+                                ]
+                            }
+                        ]
+                    }
+                ],
+                "key": "Resistance",
+                "label": "PF2E.SpecificRule.Equipment.ChannelProtectionAmuletLabel",
+                "type": "custom",
```

Comment:
Is this just so it works against either? I should be Vitality and Void I'd imagine

Reply:
It's not general resistance against either, only resistance to heal or harm.